.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "public/themes"]
 	path = public/themes
 	url = git://github.com/openSUSE/opensuse-themes.git
+
+[submodule "public/chameleon"]
+	path = public/chameleon
+	url = git@github.com:openSUSE-zh/chameleon.git
```

Comment:
I suggest to have a new repository. It has a different build system, using Gulp, SaSS and NPM. If it is a separate project, users can submit issues directly to it instead of add a tag "chameleon". It can also have its README show on the home page. Maybe this is the better way for Github.